Module tests of the parser.rs not working

I tried test the parse tests:

`cargo test --package bitcoin_scripts --lib --features all -- parser::test --nocapture `

The compiler fails and returns error:

```
error[E0599]: no method named `compile` found for enum `Concrete` in the current scope
   --> scripts/src/parser.rs:165:134
    |
165 |           ($($arg:tt)*) => (LockScript::from(miniscript::policy::Concrete::<bitcoin::PublicKey>::from_str(&format!($($arg)*)).unwrap().compile::<...
    |                                                                                                                                        ^^^^^^^ method not found in `Concrete<bitcoin::PublicKey>`
``` 

For some reason, the macro doesn't recognize the compile method.

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@ target/
 **/*.rs.bk
 
 .idea
+.vscode
 
 *.swp
 

--- a/scripts/Cargo.toml
+++ b/scripts/Cargo.toml
@@ -17,7 +17,7 @@ exclude = []
 amplify = "3.12.1"
 bitcoin = "=0.28.1" # We pin this version due to an upstream bug. See setails: <https://github.com/BP-WG/descriptor-wallet/pull/40>
 secp256k1 = { version = "0.22.1", features = ["global-context"] } # Remove after bitcoin #922
-miniscript = { version = "7.0.0", optional = true }
+miniscript = { version = "7.0.0", features=["compiler"], optional = true }
 strict_encoding = "0.8.0"
 serde_crate = { package = "serde", version = "1", features = ["derive"], optional = true }
 serde_with = { version = "1.8", optional = true }

